siac/hostcmd.go

```diff
@@ -117,16 +117,30 @@ sector may impact host revenue.`,
 // Prints info about the host and its storage folders.
 func hostcmd() {
 	hg := new(api.HostGET)
-	err := getAPI("/host", &hg)
+	err := getAPI("/host", hg)
 	if err != nil {
 		die("Could not fetch host settings:", err)
 	}
 	sg := new(api.StorageGET)
-	err = getAPI("/storage", &sg)
+	err = getAPI("/storage", sg)
 	if err != nil {
 		die("Could not fetch storage info:", err)
 	}
 
+	// Determine the competitive price string.
+	ah := new(api.ActiveHosts)
+	var competitivePrice string
+	err = getAPI("/renter/hosts/active?numhosts=24", ah)
+	if err != nil {
+		competitivePrice = "Unavailable"
+	} else {
+		var sum types.Currency
+		for _, host := range ah.Hosts {
+			sum = sum.Add(host.StoragePrice)
+		}
+		competitivePrice = currencyUnits(sum.Div64(26).Mul(modules.BlockBytesPerMonthTerabyte)) // 24 hosts were selected, but the division was 26 to lowball the average slightly.
```

Comment:
the call might not return 24 hosts. Should divide by number of hosts and then `MulFloat` by some fudge factor.
As a consequence, you need to avoid divide-by-zero by checking for `len(ah.Hosts) == 0` in the above case. I recommend:
```go
if err != nil || len(ah.Hosts) == 0 {
    competitivePrice = "Unavailable"
} else {
   // ...
}
```